packages/@aws-cdk/aws-events-targets/README.md

```diff
@@ -23,7 +23,59 @@ Currently supported are:
 * Queue a Batch job
 * Make an AWS API call
 * Put a record to a Kinesis stream
+* Invoke an API Gateway endpoint
 
 See the README of the `@aws-cdk/aws-events` library for more information on
 EventBridge.
 
+## Invoke an API Gateway endpoint
+
+Use API Gateway as a target for AWS EventBridge event rules:
```

Comment:
I think it would be good to have an example of:
> Invoke a customer-specified REST API hosted in API Gateway in an event-driven manner based on AWS or third-party events.
> Invoke an endpoint periodically based on a schedule.

per the [documentation(https://docs.aws.amazon.com/eventbridge/latest/userguide/targets-apigateway.html)